source/_components/light.limitlessled.markdown

```diff
@@ -80,3 +80,129 @@ Refer to the [light]({{site_root}}/components/light/) documentation for general
 ### {% linkable_title Initialization & Synchronization %}
 
 When starting Home Assistant, your LimitlessLED bulbs will be set to known default values. This ensures a consistent user interface and uninterrupted turning on/off. If you control your LimitlessLED lights via the MiLight mobile application or other means while Home Assistant is running, Home Assistant can not track those changes and you may observe unexpected behavior. This is due to a LimitlessLED limitation.
+
+### {% linkable_title Example Setup and Configuration %}
+
+## General Overview
+
+1. Install the MiLight phone app
+1. Connect the bridge to your network/phone
+1. Link bulbs to one of 4 groups on phone app
+1. Add config to HomeAssistant's configuration.yaml file
+
+### Install MiLight app
+
+Install from your store of choice. I used v3.7 from the Google Play store.
+
+### Connect the v6 bridge to your phone/network
+
+1. connect phone to home WiFi
+2. open MiLight app
+3. click + to go to configuration
+4. smart link
+5. enter home WiFi ssid and password
+6. press reset button once on bottom briefly, the two lights will start flashing
+7. press Start Configuration button on app
+7. it should say 'Success'
+8. lights will go solid then one will slow flashing
+9. press back on the app 
+10. press 'Searching for device' at bottom of screen
+11. your bridge should now be visible in the list, click on it to get into the configuration
+
+If you want to get to the webmin console for your bridge, use 'Advanced IP Scanner' program or similar network scanner to locate the IP address. username: admin, password: admin. It's a good idea to go in in there and set a static IP address for your bridge.
```

Comment:
**Spelling & Grammar**:
- Not quite sure, suggested: `webmin` -> `Webmin`
- `console for your bridge` -> `console on your bridge`
- Lack of context: `username: admin, password: admin` -> Maybe: `Login with the default username "admin" and password "admin".` (Or something similar).
- Unessecary words: `It's a good idea to go in in there and set a static IP address for your bridge.` -> `It's a good idea to set a static IP address for your bridge.`